BUILD.md

```diff
@@ -75,59 +105,39 @@ The shared library in the release Nuget(s) and the Python wheel may be installed
 |Linux        | NO           | YES(gcc>=4.8)    | Not tested       |
 |Mac OS X     | NO           | Not tested       | YES (Minimum version required not ascertained)|
 
-## System Requirements
-For other system requirements and other dependencies, please see [this section](./README.md#system-requirements-pre-requisite-dependencies).
 
 ---
-# Common Build Instructions
-|Description|Command|Additional description|
+## Common Build Instructions
+|Description|Command|Additional details|
 |-----------|-----------|-----------|
 |**Basic build**|build.bat (Windows)<br>./build.sh (Linux)||
 |**Debug build**|--config RelWithDebInfo|Debug build|
 |**Use OpenMP**|--use_openmp|OpenMP will parallelize some of the code for potential performance improvements. This is not recommended for running on single threads.|
 |**Build using parallel processing**|--parallel|This is strongly recommended to speed up the build.|
 |**Build Shared Library**|--build_shared_lib||
-|**Build Python wheel**|--build_wheel||
-|**Build C# and C packages**|--build_csharp||
-|**Build WindowsML**|--use_winml<br>--use_dml<br>--build_shared_lib|WindowsML depends on DirectML and the OnnxRuntime shared library.|
-|**Build Java package**|--build_java|Creates an onnxruntime4j.jar in the build directory, implies `--build_shared_lib`|
-
-
-# Additional Build Instructions
-The complete list of build options can be found by running `./build.sh (or .\build.bat) --help`
-
-* [ONNX Runtime Server (Linux)](#Build-ONNX-Runtime-Server-on-Linux)
-
-**Execution Providers**
-* [NVIDIA CUDA](#CUDA)
-* [NVIDIA TensorRT](#TensorRT)
-* [Intel DNNL/MKL-ML](#DNNL-and-MKLML)
-* [Intel nGraph](#nGraph)
-* [Intel OpenVINO](#openvino)
-* [Android NNAPI](#Android-NNAPI)
-* [Nuphar Model Compiler](#Nuphar)
-* [DirectML](#DirectML)
-* [ARM Compute Library](#ARM-Compute-Library)
-* [Rockchip RKNPU](#RKNPU)
-
-**Options**
-* [OpenMP](#OpenMP)
-* [OpenBLAS](#OpenBLAS)
-* [DebugNodeInputsOutputs](#DebugNodeInputsOutputs)
-
-**Architectures**
-* [x86](#x86)
-* [ARM](#ARM)
-* [Android](#Android)
+
+### APIs and Language Bindings
+|API|Command|Additional details|
+|-----------|-----------|-----------|
+|**Python**|--build_wheel||
+|**C# and C packages**|--build_csharp||
+|**WindowsML**|--use_winml<br>--use_dml<br>--build_shared_lib|WindowsML depends on DirectML and the OnnxRuntime shared library|
+|**Java**|--build_java|Creates an onnxruntime4j.jar in the build directory, implies `--build_shared_lib`|
```

Comment:
Compiling the Java API requires the user to have [gradle](https://gradle.org/) installed, at version 6.1 or later, in addition to the usual requirements.

Reply:
Should I add that on to the "additional details" column for Java?

Reply:
Yeah, that's probably the best place for it.